ci/jenkins/Jenkinsfile_centos_cpu

```diff
@@ -37,14 +37,16 @@ core_logic: {
     custom_steps.compile_centos7_cpu('centos7_cpu'),
     custom_steps.compile_centos7_cpu_make('centos7_cpu_make'),
     custom_steps.compile_centos7_cpu_mkldnn(),
+    custom_steps.compile_static_cd_cpu('centos7_cpu_cd'),
```

Comment:
The new test is a more elaborate version of `custom_steps.compile_static_python_cpu_cmake()` isn't it? If so, let's remove the  `custom_steps.compile_static_python_cpu_cmake()` in favor of the new approach.

Reply:
Done! Also removed custom_steps.compile_static_python_gpu_cmake() that was running cu92 in favor of new stage running cu102.